scripts/build_scripts/utils.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
```

Comment:
Nearly a direct copy of the C++ SDK version, with the removal of all of the `vcpkg` functions.